src/components/MoviesComponent/index.jsx

```diff
@@ -0,0 +1,13 @@
+// import React from 'react';
+// import { movies } from '../../components/MoviesComponent/data.json';
+
+// export const Movies = () => {
```

Comment:
For now it's fine, by consider avoiding leaving commented code in your PR or repo in general.